deploy-board/deploy_board/webapp/capacity_views.py

```diff
@@ -109,11 +109,11 @@ def post(self, request, name, stage):
         logger.info("Post to capacity with data {0}".format(request.body))
         data = json.loads(request.body)
         hosts = data.get('hosts')
-        if hosts:
+        if hosts is not None:
```

Comment:
can request.body ever be None? yay python...

Reply:
Not in my test cases. I was thinking of removing the check, but it doesn't hurt to be extra cautious.

Reply:
Actually if `request.body` is `{}`, then `hsots` is `None`.